spec/models/school_data_file_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'rails_helper'
+
+RSpec.describe SchoolDataFile, type: :model do
+  describe '#schools' do
+    let(:datafile) { file_fixture('school_data.csv') }
```

Comment:
This is personal taste, but I'm not a fan of fixtures, because:

1. If you're reading this spec, you need to open another file to see what's in the input data.
2. In the fixture data, it's not immediately obvious which rows correspond to which scenarios you're testing. For instance, I can see there are 13 rows of data but only 7 are imported. I'm guessing a bunch are filtered out but it's requires quite a bit of jumping about to figure this out.

An alternative could be to generate a temp file containing just the data you want for the specific scenario. You could use a helper that takes a string and generates a temp file. This way, you could use the `it` string to be more explicit about the scenario you're testing with that particular row of data.

Reply:
As discussed, defer until later.